src/components/messenger/list/conversation-item.tsx

```diff
@@ -124,10 +125,24 @@ export class ConversationItem extends React.Component<Properties> {
     }
   }
 
+  get dataVariant() {
```

Comment:
Isn't it possible to be both 'unread' and 'active' at the same time? This seems to only allow for a single one.

Reply:
how do you do that, since `dataVariant` is a string? Can it be an array? 

Otherwise we can use different identifies (eg. `active = 'true'/'false'`, `unread = true/false`. So their styling won't overlap. Wdyt?

Reply:
done